### PR TITLE
[PR] Feature/issue 233  new feature new role create and core usage

### LINKED ISSUE #122: [USERS] New role : Adders
**Si la requête est associée à un problème, merci de le préciser.**
Ø

**Description de la solution souhaitée**
Le but de ce role est de permettre à un utilisateur de remplir le formulaire. Il ne peut pas consulter les data et il ne peut pas les modifier.

**Piste de résolution**
Ø

**Informations supplémentaires**
L'objectif est de distribuer l'appli à des gens externes au CICR (eg les admin des pages fb)

### LINKED ISSUE #143: [USERS] Super Admin
**Si la requête est associée à un problème, merci de le préciser.**
Il peut tout voir et tout gérer qu'importe la team :

Gestion des équipes (add/modif/delete/ …)
Ajout d'utilisateurs aux équipes
Gestion des fields des équipes
Création de nouveaux utilisateurs
Gérer les data (consulter + modifier + delete …)
Modification des rôles
Ne fait pas parti d'une équipe, donc pas de possibilité d'ajouter des nouvelles data ?

**Description de la solution souhaitée**
0

**Piste de résolution**
0

**Informations supplémentaires**
0

### LINKED ISSUE #233: [NEW FEATURE] New Role create and core usage
**If the request is associated with a problem, please specify it.**
-> new User create view with checkboxs
-> new table with all the rights of the roles
-> in policies, check the rights one per one (and not the name)

### LINKED ISSUE #242: [OTHER] Roles crew linked by filtering groups of roles in the dropdown
**What is the purpose of this issue ?**
Making roles pseudo-cew-linked

: by filtering the roles by teams inside the dropdown

### PR DESCRIPTION
**Pull request by mathisfchx**
### Description
ROLE V2

## Issues fixed
List of related issues that are fixed by this PR :

- Closed #122
- Closed #143
- Closed #242
- Closed #233

### Feature Checklist
- [x] Cytoscape
- [x] Person/Index
- [x] Person/Filter
- [x] Person/Create
- [x] Person/Edit
- [x] Person/Delete
- [x] Relation/Index
- [x] Relation/Filter
- [x] Relation/Create
- [x] Relation/Edit
- [x] Relation/Delete
- [x] Duplicates/Index
- [ ] Duplicates/Filter
- [ ] Duplicates/Create
- [ ] Duplicates/Edit
- [x] ManageList/Delete
- [x] ManageList/Create
- [x] ManageList/Edit
- [x] ManageField/Index
- [x] ManageField/Create
- [x] ManageField/Edit
- [x] ManageField/Delete
- [x] ApiList
- [x] Crews
- [x] Users

### CODE REVIEW

File: src/resources/views/roles/index.blade.php [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    <div class="py-12">
+        <div class="max-w-7xl mx-auto sm:px-6 lg:px-8">
+            <div class="block mb-8">
+                <a href="{{URL::previous() }}"
+                   class="bg-gray-200 hover:bg-gray-300 text-black font-bold py-2 px-4 rounded">Back</a>
+                @can('create', \App\Models\Role::class)
+                    <a href="{{ route("roles.create") }}"
+                       class="bg-green-500 hover:bg-green-700 text-white font-bold py-2 px-4 rounded">New role</a>
+                @endcan
+                @if (env("APP_DEBUG"))
```
Comment (lduf) [inline src/resources/views/roles/index.blade.php:18]: ❤️

**Review by lduf — CHANGES_REQUESTED** [review]
Please, consider the following :

**Comment by lduf** [comment]
fixes #248

**Comment by lduf** [comment]
Add a way to match 
route_controller and route_policy

-> add an attribute into model

If the attribute is defined then use the matching options. Else use the controller_route

**Review by mathisfchx — COMMENTED** [review]
🌍

**Review by lduf — APPROVED** [review]
🌍

**Pull request merged**